var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,8 +32,11 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    url 	 = "http://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.6.tar.bz2"
+    list_url = "https://ftp.heanet.ie/disk1/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    list_depth = 2
```

Comment:
I'm not sure whether or not we want `list_url` here. Even if we do, `list_url` should be a directory, and `list_depth` should not be 2. See the documentation for more details on what these attributes actually do.